spec/support/controller_macros.rb

```diff
@@ -0,0 +1,9 @@
+module ControllerMacros
```

Comment:
Why do we need this file? What does it do?

Reply:
Just in case, for the future tests.

Deleted! I will add it later, if necessary